src/base/rss/private/rss_parser.cpp

```diff
@@ -603,6 +604,8 @@ void Parser::parseRssArticle(QXmlStreamReader &xml)
             else if (name == QLatin1String("enclosure")) {
                 if (xml.attributes().value("type") == QLatin1String("application/x-bittorrent"))
                     article[Article::KeyTorrentURL] = xml.attributes().value(QLatin1String("url")).toString();
+                else if (xml.attributes().value("type").isEmpty())
```

Comment:
The following should be more efficient:
```c++
else if (!xml.hasAttribute("type"))
```

Reply:
That would miss out on cases where you have the attribute, but it's empty since the attribute is mandatory but the value isn't.

Reply:
Ok.